pkg/job/status.go

```diff
@@ -82,9 +82,8 @@ type Status struct {
 	// started yet"
 	StartTime time.Time
 
-	// EndTime indicates when the job started. This value should be ignored if
-	// `Finished` is false.
-	EndTime time.Time
+	// EndTime indicates when the job started.
```

Comment:
```suggestion
	// EndTime indicates when the job ended.
```

Reply:
wow, that has been there for quite a while :D